app/views/search/_variable_search_results.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="pagination-summary">
+  <%= page_entries_info variables, entry_name: 'record' %>
```

Comment:
Seems like we would want this to say 'No variables found' instead of 'No records found'.